index.js

```diff
@@ -80,7 +82,28 @@ module.exports = {
   },
 
   _getAllD3Modules() {
-    return d3DepsForPackage('d3', this.parent.nodeModulesPath, this.ui);
+    var d3Deps = d3DepsForPackage('d3', this.parent.nodeModulesPath, this.ui);
+
+    var pluginNames = Object.keys(this.parent.dependencies()).filter(isD3Dependency);
```

Comment:
Because we're likely to know which plugins we're including, it might make more sense to just explicitly expose them in the config with some sort of `includes` property perhaps. 

I'm hesitant to assume all d3 plugins will have `d3-` in the name, esp. if they're 3rd party.